docs/src/pages/getting-started/installation/installation.md

```diff
@@ -20,6 +20,18 @@ yarn add @material-ui/core@next @emotion/react @emotion/styled
 
 Please note that [react](https://www.npmjs.com/package/react) >= 17.0.0 and [react-dom](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.
 
+Or if you want to use `styled-components` as a styling engine:
+
+```sh
+// with npm
+npm install @material-ui/core@next @material-ui/styled-engine-sc@next styled-components
+
+// with yarn
+yarn add @material-ui/core@next @material-ui/styled-engine-sc@next styled-components
+```
+
+Take a look on the [Styling Engine guide](/customization/styling-engine/) for more information about how to configure this.
```

Comment:
```suggestion
Take a look at the [Styling Engine guide](/customization/styling-engine/) for more information about how to configure this.
```

Not sure though.

Also: What is "this" referring to? Implicit reference should generally be avoided in technical writing. Learning something new is already a lot of cognitive load so adding more burden by having to resolve references in your head makes learning harder. See ["Reduce there is/there are"](https://developers.google.com/tech-writing/one/clear-sentences)

Reply:
Updated